.github/workflows/build-checks.yml

```diff
@@ -117,7 +117,7 @@ jobs:
         run: cmake --build . -j $(nproc) -- --keep-going
         working-directory: build
         env:
-          CLAZY_CHECKS: level2,no-rule-of-two-soft,no-non-pod-global-static,no-qproperty-without-notify,no-wrong-qevent-cast,no-qstring-allocations,no-function-args-by-value,no-copyable-polymorphic,no-ctor-missing-parent-argument,no-missing-qobject-macro,no-rule-of-three,no-returning-void-expression,no-missing-typeinfo,no-base-class-event
+          CLAZY_CHECKS: level2,no-rule-of-two-soft,no-non-pod-global-static,no-qproperty-without-notify,no-qstring-allocations,no-function-args-by-value,no-copyable-polymorphic,no-ctor-missing-parent-argument,no-rule-of-three
```

Comment:
I don't think we should use `no-returning-void-expression`. It results in too much noise in the tests and the bug it protects against (at least in their example) is very unlikely to occur in mixxx due to our code formatting rules.

Reply:
I didn't spot and true positives for `no-returning-void-expression` in the diff. Did you?

Reply:
https://github.com/mixxxdj/mixxx/blob/main/src/engine/controls/enginecontrol.cpp#L74

Reply:
unnecessary, sure, but not a bug.